sdk/core/Azure.Core/src/Shared/HttpMessageSanitizer.cs

```diff
@@ -1,140 +1,198 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+#nullable enable
 
 using System;
 using System.Collections.Generic;
 using System.Text;
 using System.Linq;
 
-namespace Azure.Core
+namespace Azure.Core;
+
+internal class HttpMessageSanitizer
```

Comment:
@m-redding as FYI

Reply:
Are you calling out the file-scoped namespace? I did that because I got an IDE-level analysis warning about it. I don't think it broke the build, though. I can revert it if you want.

Reply:
@pharring nw about the namespace - I'm just working on a feature that uses the sanitizer